features/step_definitions/hangout_steps.rb

```diff
@@ -182,4 +185,7 @@
   page.should have_field('hangout_url', with: @hangout_url)
 end
 
-
+Then(/^a separate event instance is not created$/) do
+  expect(EventInstance.where("created_at >= ?", Time.now.beginning_of_day).size)
```

Comment:
I just checked, yes the date set in first step persist, so it doesn't matter when the test is ran. Time.now would return 2014-02-07 07:01:41 +0000